### ISSUE TITLE
TRL/Alignment-Handbook torch.dtype Issues

### ISSUE THREAD

**Issue by neelsjain** [huggingface/alignment-handbook#175]
I believe that the `torch.dtype` handling is currently broken in [TRL](https://github.com/huggingface/trl/issues/1751), which is causing issues with alignment handbook. I was able to get around this by downgrading to the TRL version (0.8.2) when this repo was created.

**Comment by badmic** [comment]
0.8.2的版本下载不了

**Comment by alvarobartt** [comment]
Hi @neelsjain thanks for reporting here too! Indeed `trl` latest version will be released soon, and it already comes with the `torch_dtype` issue fixed so that it handles `str`, `torch.dtype` or `None` values. Feel free to install `trl` from source in the meantime as follows:

```bash
pip install "trl @ git+https://github.com/huggingface/trl.git@main" --upgrade
```

Let me know if that works in the meantime 🤗

**Comment by alvarobartt** [comment]
Hi @neelsjain is this already solved after upgrading `trl` to v0.9.6, or is this still an issue? Thanks in advance!

**Comment by neelsjain** [comment]
Yes, it is! I will close the issue.
